app/assets/javascripts/channels/export.js

```diff
@@ -1,37 +1,42 @@
-function subscribeExportChannel(uuid, callback) {
-  App.download = App.cable.subscriptions.create(
+function subscribeExportChannel(uuid, startExportJobFtn, completionCallback) {
```

Comment:
maybe `startingCallback` ?

just thinking about consistency with `completionCallback`

Reply:
I switched to passing in the `url` since I needed it for error reporting.